src/cli.rs

```diff
@@ -210,43 +210,50 @@ fn build_command() -> Command<'static> {
                 .takes_value(true)
                 .value_name("UNIT")
                 .possible_values(&["millisecond", "second"])
-                .help("Set the time unit to be used. Possible values: millisecond, second."),
+                .help("Set the time unit to be used. Possible values: millisecond, second.\
```

Comment:
You need to leave a space at the end here:
```suggestion
                .help("Set the time unit to be used. Possible values: millisecond, second. \
```

Otherwise, the help text reads like this:
![image](https://user-images.githubusercontent.com/4209276/176243264-b7fde7bc-17a0-4221-8cea-d264051cabee.png)